### ISSUE TITLE
SLSA provenance is too hard to find on the website

### ISSUE THREAD

**Issue by TomHennen** [slsa-framework/slsa#526]
I've heard from a couple of folks that they weren't aware that SLSA had a specification for how to exchange metadata, despite the fact that SLSA Provenance is linked from the menu.

Would it make sense to make SLSA provenance and VSAs more prominent on the SLSA website?

**Comment by MarkLodato** [comment]
Yes. This is currently blocked on #340. I think we should have an "Attestations" at the top level. (cc: @olivekl)
